apps/mobile/src/features/ConfirmationsSheet/ConfirmationsSheet.container.tsx

```diff
@@ -69,26 +85,26 @@ export const ConfirmationsSheetContainer = () => {
                 circular={false}
                 content={
                   <View alignItems="center" flexDirection="row" gap="$1">
-                    {hasSigned && <SafeFontIcon size={12} name="check" />}
+                    {(isHistoryTransaction || hasSigned) && <SafeFontIcon size={12} name="check" />}
 
                     <Text fontWeight={600} color={'$color'}>
-                      {hasSigned ? 'Signed' : 'Pending'}
+                      {isHistoryTransaction ? 'Signed' : hasSigned ? 'Signed' : 'Pending'}
```

Comment:
Would it also work to do isHistoryTransaction || hasSigned here like above?

Reply:
Good point. I originally wrote it like this becuase if we are dealing with signed tx the only state available was obvisouly only "signed". But yea, doing like above is better.